akka-docs/rst/java/http/routing-dsl/directives/misc-directives/withSizeLimit.rst

```diff
@@ -0,0 +1,20 @@
+.. _-withSizeLimit-java-:
+
+withSizeLimit
+===============
+
+Description
+-----------
+Rejects all requests which request entity size exceeds given limit. Limit given
```

Comment:
It does not reject, please see https://github.com/akka/akka/pull/20760/commits/9e364c89214d2b6f58c1e72a393836a3e033819d#r68016626